src/gi/overrides/Vast.py

```diff
@@ -0,0 +1,6 @@
+from ..module import get_introspection_module
+
+Vast = get_introspection_module('Vast')
+
+Vast.Array.__getitem__ = lambda self, i: self.get_value(i)
```

Comment:
We need to refine this a bit in the future. If i is a slice or tuple of slices, return an array viewing the current array.
But what if it is a mixture of slice and indices? I don't think the current Array api support this; the dimensions indexed with a integer shall be eliminated.